save-demo-agent/src/nativeMain/kotlin/com/saveourtool/save/demo/agent/Main.kt

```diff
@@ -0,0 +1,42 @@
+/**
+ * An entry point for save-demo-agent.
+ */
+
+package com.saveourtool.save.demo.agent
+
+import com.saveourtool.save.core.utils.ProcessBuilder
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.cio.*
+import io.ktor.server.engine.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import okio.FileSystem
+
+@Suppress("MagicNumber")
+fun main() {
+    runServerOnPort(23456).start()
```

Comment:
make port configurable via properties file?

Reply:
Will be done in next PR